ocs_ci/ocs/resources/catalog_source.py

```diff
@@ -18,7 +18,8 @@ class CatalogSource(OCP):
     methods we need to do with it.
     """
 
-    def __init__(self, resource_name="", *args, **kwargs):
+    def __init__(self, resource_name="", resource_namespace="", *args,
+                 **kwargs):
```

Comment:
I see we are missing Args in docstring here, can you please add it? Thanks

Reply:
Done